Lib/typing.py

```diff
@@ -629,7 +642,10 @@ def __init__(self, origin, params, *, inst=True, special=False, name=None):
         self._special = special
         if special and name is None:
             orig_name = origin.__name__
-            name = orig_name[0].title() + orig_name[1:]
+            if orig_name in _normalize_alias:
+                name = _normalize_alias[orig_name]
```

Comment:
```python
name = _normalize_alias.get(orig_name, orig_name)
```
?

Reply:
Hm, interesting suggestion, let's go with it.